web/src/utils/state/atoms.ts

```diff
@@ -46,8 +46,10 @@ export const feedbackCardCollapsedNumberAtom = atom(0);
 
 export const colorblindModeAtom = atomWithStorage('colorblindModeEnabled', false);
 
-export const dataSourcesCollapsedBarBreakdown = atom(true);
+export const dataSourcesCollapsedBarBreakdown = atom<boolean | undefined>(true);
 
-export const dataSourcesCollapsedBreakdown = atom(true);
+export const dataSourcesCollapsedBreakdown = atom<boolean | undefined>(true);
 
-export const dataSourcesCollapsedEmission = atom(true);
+export const dataSourcesCollapsedEmission = atom<boolean | undefined>(true);
+
+export const estimationCardCollapsed = atom<boolean | undefined>(undefined);
```

Comment:
Why is it necessary to have `atom<boolean | undefined>` and not just `atom<boolean>`?

Reply:
I didn't want it to have a state from the begging so it can be used for both the estimation card and the outage card, but perhaps it's better to split them into two and just use a boolean value.